test/parallel/test-os-eol.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const os = require('os');
+
+const eol = common.isWindows ? '\r\n' : '\n';
+
+assert.strictEqual(eol, os.EOL);
```

Comment:
Assertion parameters should be `actual, expected` so I think these should be the other way around.